docs/en/latest/admin-api.md

```diff
@@ -1497,6 +1497,102 @@ HTTP/1.1 200 OK
 {"key":"\/apisix\/secrets\/vault\/test2","value":{"id":"vault\/test2","token":"apisix","prefix":"apisix","update_time":1669625828,"create_time":1669625828,"uri":"http:\/\/xxx\/get"}}
 ```
 
+#### When Secret Manager is AWS
+
+| Parameter | Required | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| access_key_id | True | string | AWS Access Key ID |  |
+| secret_access_key | True | string | AWS Secret Access Key |  |
+| session_token | False | string | Temporary access credential information |  |
+| region | False | string | AWS Region |  |
+| endpoint_url | False | URI | AWS Secret Manager URL | https://secretsmanager.{region}.amazonaws.com |
```

Comment:
The example column seems redundant. I would suggest using the following table instead:

```suggestion
| Parameter         | Required | Type   | Description                                                                 |
| ----------------- | -------- | ------ | --------------------------------------------------------------------------- |
| access_key_id     | True     | string | AWS Access Key ID                                                           |
| secret_access_key | True     | string | AWS Secret Access Key                                                       |
| session_token     | False    | string | Temporary access credential information                                     |
| region            | False    | string | AWS Region                                                                  |
| endpoint_url      | False    | URI    | AWS Secret Manager URL (e.g: https://secretsmanager.{region}.amazonaws.com) |
```

Reply:
I think there is no need to remove the example field, as the vault document also contains this field, and docs consistency should be maintained.

Reply:
Fixed.